GitHub.Collectors/Model/DefaultCollector.cs

```diff
@@ -22,13 +25,11 @@ public class DefaultCollector : ICollector
         public static readonly HttpResponseSignature UserNotFoundResponse = new HttpResponseSignature(HttpStatusCode.NotFound, "Not Found");
         public static readonly HttpResponseSignature ResourceNotAccessibleByIntegrationResponse = new HttpResponseSignature(HttpStatusCode.Forbidden, "Resource not accessible by integration");
 
-        public const string OrganizationInstanceRecordType = "GitHub.OrgIstance";
-        public const string UserInstanceRecordType = "GitHub.UserInstance";
-
         protected FunctionContext FunctionContext { get; private set; }
         protected GitHubHttpClient HttpClient { get; private set; }
         protected List<IRecordWriter> RecordWriters { get; private set; }
         protected ICache<RepositoryItemTableEntity> Cache { get; private set; }
+        protected ICache<PointCollectorTableEntity> PointCache { get; private set; }
```

Comment:
>private set; [](http://example.com/codeflow?start=70&length=12)

Nit: if these are only set in the constructor, you can remove "private set;" parts.